### DISCUSSION TITLE
How can I extract/calculate point cloud from mesh?

### DISCUSSION THREAD

**Discussion by K-M-Ibrahim-Khalilullah** [pyvista/pyvista#1755] [Q&A]
I want to get point cloud mesh? How can calculate it?
Thanks

**Comment by banesullivan** [comment]
To make a point cloud from a mesh, simply: `pv.PolyData(mesh.points)`

```py
import pyvista as pv

mesh = pv.Sphere()

points = pv.PolyData(mesh.points)

p = pv.Plotter(shape=(1,2))
p.add_mesh(mesh)
p.subplot(0,1)
p.add_mesh(points, render_points_as_spheres=True)
p.link_views()
p.show()
```
![Screen Shot 2021-11-19 at 10 27 38 AM](https://user-images.githubusercontent.com/22067021/142665435-ebd6b41f-d78a-49b0-925c-3e01d504df33.png)

**Reply by banesullivan** [reply]
If the source mesh has point data, be sure to do:

```py
points = pv.PolyData(mesh.points)
points.point_data.update(mesh.point_data)
```
